gulpfile.babel.js

```diff
@@ -1,6 +1,5 @@
 import chug from 'gulp-chug';
 import gulp from 'gulp';
-import upath from 'path';
```

Comment:
Is it the only place `upath` is used? If so, we can delete it from `package.json` as well :)

Reply:
No, it's used in the other `gulpfile`s :smile: